### PR TITLE
K8SPS-54: Add restores

### PR DESCRIPTION
**Pull request by egegunes**
This is a basic implementation of restores. It only supports restores from S3, GCS and Azure Object Storage.

Once the restore CR created:

1. The operator pauses the cluster (terminates MySQL and Orchestrator pods).
2. A Job for restore is created. First MySQL pod's datadir (i.e `datadir-cluster1-mysql-0`) is mounted to the Pod controlled by the Job. All contents of datadir is deleted and replaced by the contents of the backup.
3. If restore succeeds, the operator resumes the cluster.

### CODE REVIEW

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| config | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-80/e20b930/config.log) |
| init-deploy | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-80/e20b930/init-deploy.log) |
| monitoring | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-80/e20b930/monitoring.log) |
| semi-sync | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-80/e20b930/semi-sync.log) |
| service-per-pod | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-80/e20b930/service-per-pod.log) |
| scaling | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-80/e20b930/scaling.log) |
| sidecars | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-80/e20b930/sidecars.log) |
| users | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-80/e20b930/users.log) |
| We run 8 out of 9|

commit: https://github.com/percona/percona-server-mysql-operator/pull/80/commits/e20b930751052ab056acfbb1898ca1ede3ceda5d
image: `perconalab/percona-server-mysql-operator:PR-80-e20b930`

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| config | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-80/e20b930/config.log) |
| init-deploy | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-80/e20b930/init-deploy.log) |
| monitoring | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-80/e20b930/monitoring.log) |
| semi-sync | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-80/e20b930/semi-sync.log) |
| service-per-pod | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-80/e20b930/service-per-pod.log) |
| scaling | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-80/e20b930/scaling.log) |
| sidecars | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-80/e20b930/sidecars.log) |
| users | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-80/e20b930/users.log) |
| demand-backup | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-80/e20b930/demand-backup.log) |
| We run 9 out of 9|

commit: https://github.com/percona/percona-server-mysql-operator/pull/80/commits/e20b930751052ab056acfbb1898ca1ede3ceda5d
image: `perconalab/percona-server-mysql-operator:PR-80-e20b930`

**Review by hors — CHANGES_REQUESTED** [review]

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| config | [failed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-80/912eee2/config.log) |
| init-deploy | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-80/912eee2/init-deploy.log) |
| monitoring | [failed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-80/912eee2/monitoring.log) |
| semi-sync | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-80/912eee2/semi-sync.log) |
| service-per-pod | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-80/912eee2/service-per-pod.log) |
| scaling | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-80/912eee2/scaling.log) |
| sidecars | [failed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-80/912eee2/sidecars.log) |
| users | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-80/912eee2/users.log) |
| demand-backup | [failed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-80/912eee2/demand-backup.log) |
| We run 9 out of 9|

commit: https://github.com/percona/percona-server-mysql-operator/pull/80/commits/912eee28aa1fb90de632e5ef4bcc3dc0d025f43a
image: `perconalab/percona-server-mysql-operator:PR-80-912eee2`

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| config | [failed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-80/2845ce8/config.log) |
| init-deploy | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-80/2845ce8/init-deploy.log) |
| monitoring | [failed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-80/2845ce8/monitoring.log) |
| semi-sync | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-80/2845ce8/semi-sync.log) |
| service-per-pod | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-80/2845ce8/service-per-pod.log) |
| scaling | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-80/2845ce8/scaling.log) |
| sidecars | [failed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-80/2845ce8/sidecars.log) |
| users | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-80/2845ce8/users.log) |
| demand-backup | [failed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-80/2845ce8/demand-backup.log) |
| We run 9 out of 9|

commit: https://github.com/percona/percona-server-mysql-operator/pull/80/commits/2845ce8993cc84d3a01c88d88ff21b8851dc1ab6
image: `perconalab/percona-server-mysql-operator:PR-80-2845ce8`

File: build/run-restore.sh [hunk 1]
```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
+DATADIR=${DATADIR:-/var/lib/mysql}
+
+run_s3() {
+	xbcloud get "${BACKUP_NAME}" --storage=s3 --s3-bucket="${S3_BUCKET}"
```
[the remark this answers is not in the document]
Comment (egegunes) [reply build/run-restore.sh]: https://github.com/percona/percona-server-mysql-operator/pull/80/commits/aa0d7bdd56a3fac9815296fd911497b560942a09

**Review by pooknull — APPROVED** [review]

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| config | [failed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-80/375407c/config.log) |
| init-deploy | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-80/375407c/init-deploy.log) |
| monitoring | [failed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-80/375407c/monitoring.log) |
| semi-sync | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-80/375407c/semi-sync.log) |
| service-per-pod | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-80/375407c/service-per-pod.log) |
| scaling | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-80/375407c/scaling.log) |
| sidecars | [failed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-80/375407c/sidecars.log) |
| users | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-80/375407c/users.log) |
| demand-backup | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-80/375407c/demand-backup.log) |
| We run 9 out of 9|

commit: https://github.com/percona/percona-server-mysql-operator/pull/80/commits/375407cf84fe06242ce74a17f12363bd2458b4ab
image: `perconalab/percona-server-mysql-operator:PR-80-375407c`

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| config | [failed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-80/5b3f0b0/config.log) |
| init-deploy | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-80/5b3f0b0/init-deploy.log) |
| monitoring | [failed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-80/5b3f0b0/monitoring.log) |
| semi-sync | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-80/5b3f0b0/semi-sync.log) |
| service-per-pod | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-80/5b3f0b0/service-per-pod.log) |
| scaling | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-80/5b3f0b0/scaling.log) |
| sidecars | [failed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-80/5b3f0b0/sidecars.log) |
| users | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-80/5b3f0b0/users.log) |
| demand-backup | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-80/5b3f0b0/demand-backup.log) |
| We run 9 out of 9|

commit: https://github.com/percona/percona-server-mysql-operator/pull/80/commits/5b3f0b09e072fdd7ba17fe46febbe5162c483429
image: `perconalab/percona-server-mysql-operator:PR-80-5b3f0b0`

**Review by tplavcic — CHANGES_REQUESTED** [review]

File: api/v1alpha1/perconaservermysqlrestore_types.go [hunk 2]
```diff
@@ -25,23 +25,32 @@ import (
 
 // PerconaServerMySQLRestoreSpec defines the desired state of PerconaServerMySQLRestore
 type PerconaServerMySQLRestoreSpec struct {
-	// INSERT ADDITIONAL SPEC FIELDS - desired state of cluster
-	// Important: Run "make" to regenerate code after modifying this file
-
-	// Foo is an example field of PerconaServerMySQLRestore. Edit perconaservermysqlrestore_types.go to remove/update
-	Foo string `json:"foo,omitempty"`
+	ClusterName string `json:"clusterName"`
+	BackupName  string `json:"backupName"`
```
[the remark this answers is not in the document]
Comment (egegunes) [reply api/v1alpha1/perconaservermysqlrestore_types.go:29]: let's create a ticket for it, I think we can leave it for next releases

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| config | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-80/b88b933/config.log) |
| init-deploy | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-80/b88b933/init-deploy.log) |
| monitoring | [failed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-80/b88b933/monitoring.log) |
| semi-sync | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-80/b88b933/semi-sync.log) |
| service-per-pod | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-80/b88b933/service-per-pod.log) |
| scaling | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-80/b88b933/scaling.log) |
| sidecars | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-80/b88b933/sidecars.log) |
| users | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-80/b88b933/users.log) |
| demand-backup | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-80/b88b933/demand-backup.log) |
| We run 9 out of 9|

commit: https://github.com/percona/percona-server-mysql-operator/pull/80/commits/b88b9336dfbd6023b6658c877231e937393eb17d
image: `perconalab/percona-server-mysql-operator:PR-80-b88b933`

File: build/run-restore.sh [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+	tmpdir=$(mktemp --directory)
+
+	case ${STORAGE_TYPE} in
+		"s3") run_s3 | extract ${tmpdir} ;;
+		"gcs") run_gcs | extract ${tmpdir} ;;
+		"azure") run_azure | extract ${tmpdir} ;;
+	esac
+
+	xtrabackup --prepare --rollback-prepared-trx --target-dir=${tmpdir}
+	xtrabackup --datadir=${DATADIR} --move-back --force-non-empty-directories --target-dir=${tmpdir}
```
Comment (hors) [inline build/run-restore.sh:46]: we can use datadir
```suggestion
	xtrabackup --datadir= /datadir --move-back --force-non-empty-directories --target-dir=${tmpdir}
```

File: build/run-restore.sh [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+
+	xbstream -xv -C ${targetdir} --parallel=${PARALLEL}
+}
+
+main() {
+	echo "Starting restore ${RESTORE_NAME}"
+	echo "Restoring to backup ${BACKUP_NAME}: ${BACKUP_DEST}"
+
+	rm -rf ${DATADIR}/*
+	tmpdir=$(mktemp --directory)
```
Comment (hors) [inline build/run-restore.sh:37]:
```suggestion
	tmpdir=$(mktemp --directory /datadir/pxc_sst_XXXX)
```

**Review by hors — CHANGES_REQUESTED** [review]

File: build/run-restore.sh (hunk 3 above)
Comment (egegunes) [reply build/run-restore.sh:46]: we don't have `/datadir` path in the container, we mount the pvc to `/var/lib/mysql`

File: build/run-restore.sh (hunk 4 above)
Comment (egegunes) [reply build/run-restore.sh:37]: https://github.com/percona/percona-server-mysql-operator/pull/80#discussion_r902573430

File: api/v1alpha1/perconaservermysql_types.go [hunk 5]
```diff
@@ -410,6 +410,11 @@ func (cr *PerconaServerMySQL) CheckNSetDefaults(serverVersion *platform.ServerVe
 		return errors.New("Orchestrator size must be 3 or greater and an odd number for raft setup")
 	}
 
+	if cr.Spec.Pause {
```
Comment (tplavcic) [inline api/v1alpha1/perconaservermysql_types.go:413]: Please add pause and allowUnsafeConfigurations to default cr.yaml if they should be supported:
This is how we have them in PXC operator:
```
  allowUnsafeConfigurations: false
#  pause: false
```

File: build/run-restore.sh (hunk 4 above)
Comment (hors) [reply build/run-restore.sh:37]: ok, but please do not use tmp, please use` /var/lib/mysql` local storage can be small for the backup

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| config | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-80/d31b54f/config.log) |
| init-deploy | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-80/d31b54f/init-deploy.log) |
| monitoring | [failed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-80/d31b54f/monitoring.log) |
| semi-sync | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-80/d31b54f/semi-sync.log) |
| service-per-pod | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-80/d31b54f/service-per-pod.log) |
| scaling | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-80/d31b54f/scaling.log) |
| sidecars | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-80/d31b54f/sidecars.log) |
| users | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-80/d31b54f/users.log) |
| demand-backup | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-80/d31b54f/demand-backup.log) |
| We run 9 out of 9|

commit: https://github.com/percona/percona-server-mysql-operator/pull/80/commits/d31b54fb25b684982f94f026b7839adaf121bdf0
image: `perconalab/percona-server-mysql-operator:PR-80-d31b54f`

File: api/v1alpha1/perconaservermysql_types.go (hunk 5 above)
Comment (egegunes) [reply api/v1alpha1/perconaservermysql_types.go:413]: https://github.com/percona/percona-server-mysql-operator/pull/80/commits/bc1843afdc8568e862a948e5ac2e7afe1d6335cf

**Review by hors — CHANGES_REQUESTED** [review]

**Review by pooknull — CHANGES_REQUESTED** [review]

File: cmd/sidecar/main.go [hunk 6]
```diff
@@ -243,18 +250,36 @@ func backupHandler(w http.ResponseWriter, req *http.Request) {
...
+			return
+		}
+
+		if err := xbcloud.Wait(); err != nil {
+			log.Error(err, "failed waiting for xbcloud to finish")
+			http.Error(w, "backup failed", http.StatusInternalServerError)
+			return
+		}
+
+		xbcloudDone <- struct{}{}
```
[the remark this answers is not in the document]
Comment (egegunes) [reply cmd/sidecar/main.go:276]: very good catch, thank you: https://github.com/percona/percona-server-mysql-operator/pull/80/commits/ab14c841766326977d96feeeff602b4572d9cb3c

File: deploy/operator.yaml [hunk 7]
```diff
@@ -41,7 +41,7 @@ spec:
             fieldRef:
               apiVersion: v1
               fieldPath: metadata.namespace
-        image: perconalab/percona-server-mysql-operator:main
+        image: perconalab/percona-server-mysql-operator:k8sps-54
```
[the remark this answers is not in the document]
Comment (egegunes) [reply deploy/operator.yaml]: https://github.com/percona/percona-server-mysql-operator/pull/80/commits/e60f5aced8c1ce433300c1d764df6360b22ed4bb

File: deploy/restore.yaml [hunk 8]
```diff
@@ -1,7 +1,7 @@
 apiVersion: ps.percona.com/v1alpha1
 kind: PerconaServerMySQLRestore
 metadata:
-  name: restore1
+  name: restore2
 spec:
   clusterName: cluster1
-  backupName: backup1
+  backupName: backup3
```
[the remark this answers is not in the document]
Comment (egegunes) [reply deploy/restore.yaml]: https://github.com/percona/percona-server-mysql-operator/pull/80/commits/e00441dfa90185d1f845256126c67dd893eaf0eb

**Review by pooknull — APPROVED** [review]

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| config | [failed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-80/bc1843a/config.log) |
| init-deploy | [failed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-80/bc1843a/init-deploy.log) |
| monitoring | [failed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-80/bc1843a/monitoring.log) |
| semi-sync | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-80/bc1843a/semi-sync.log) |
| service-per-pod | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-80/bc1843a/service-per-pod.log) |
| scaling | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-80/bc1843a/scaling.log) |
| sidecars | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-80/bc1843a/sidecars.log) |
| users | [failed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-80/bc1843a/users.log) |
| We run 8 out of 9|

commit: https://github.com/percona/percona-server-mysql-operator/pull/80/commits/bc1843afdc8568e862a948e5ac2e7afe1d6335cf
image: `perconalab/percona-server-mysql-operator:PR-80-bc1843a`

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| config | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-80/e00441d/config.log) |
| init-deploy | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-80/e00441d/init-deploy.log) |
| monitoring | [failed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-80/e00441d/monitoring.log) |
| semi-sync | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-80/e00441d/semi-sync.log) |
| service-per-pod | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-80/e00441d/service-per-pod.log) |
| scaling | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-80/e00441d/scaling.log) |
| sidecars | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-80/e00441d/sidecars.log) |
| users | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-80/e00441d/users.log) |
| demand-backup | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-80/e00441d/demand-backup.log) |
| We run 9 out of 9|

commit: https://github.com/percona/percona-server-mysql-operator/pull/80/commits/e00441dfa90185d1f845256126c67dd893eaf0eb
image: `perconalab/percona-server-mysql-operator:PR-80-e00441d`

**Review by tplavcic — APPROVED** [review]

**Pull request merged**